src/test/java/com/algosenpai/app/chapters/chapter1/BubbleSortPassesQuestionTest.java

```diff
@@ -0,0 +1,50 @@
+package com.algosenpai.app.chapters.chapter1;
+
+import com.algosenpai.app.logic.chapters.chapter1.BubbleSortPassesQuestion;
+import com.algosenpai.app.logic.models.QuestionModel;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class BubbleSortPassesQuestionTest {
+
+    /**
+     * Creates my own BubbleSortPassesQuestion then checks if the parameters are correct.
+     */
+    @Test
+    public void questionTesting() {
+        int[] arr = {5,4,3,2,1};
+        ArrayList<Integer> array = new ArrayList<>();
+        for (int value : arr) {
+            array.add(value);
+        }
+        ArrayList<String> myWrongAnswers = new ArrayList<>();
+        myWrongAnswers.add("4, 3, 2, 1, 5");
```

Comment:
Slightly confusing due to many magic numbers, you might want to add a comment to explain